tests/utils_tests/test_numberformat.py

```diff
@@ -49,6 +49,10 @@ def test_large_number(self):
         self.assertEqual(nformat(-1 - int_max, '.'), most_max.format('-', '9'))
         self.assertEqual(nformat(-2 * int_max, '.'), most_max2.format('-'))
 
+    def test_float_numbers(self):
+        # The following differs from the Decimal('3.') case because the latter behaves more like an integer
```

Comment:
I should have explained more clearly.

I wanted to document the non-obvious difference between passing `numberformat.format()` a float `3.` and a `Decimal(3.)`. In the decimal case, it seems that the "fractional" part is zero, and the result is identical to formatting an int.

```python
>>> nformat(3., '.')
'3.0'
>>> nformat(Decimal('3.'), '.')  # instantiate Decimal with a string
'3'
>>> nformat(Decimal(3.), '.')  # instantiate Decimal with a float
'3'
>>> nformat(3, '.')
'3'
```